src/ACF/Block_Config.php

```diff
@@ -27,7 +27,9 @@ abstract class Block_Config {
 
 	abstract protected function add_fields();
 
-	abstract protected function add_settings();
+	protected function add_settings() {
+		//overrite in sub class to add settings
```

Comment:
Could there be a case where we'd want a no-op `add_fields()`, too? Seems like `add_fields()` and `add_settings()` should be consistent.

Reply:
agreed